app/Models/Post.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Post extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'posts';
+    /**
+     * [$dates description]
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+    /**
+     * [$fillable description]
+     * @var [type]
+     */
+    protected $fillable = [
+        'user_id',
+        'topic_id',
+        'title',
+        'slug',
+        'body',
+        'type',
+        'created_at',
+        'updated_at',
+        'deleted_at'
+    ];
+    /**
+     * Get images: One to many
+     * @return [type] [description]
+     */
+    public function images() {
+        return $this->hasMany('App\Models\Image', 'post_id');
+    }
+    /**
+     * Get reports: One to many
+     * @return [type] [description]
+     */
+    public function reports() {
+        return $this->hasMany('App\Models\Report', 'post_id');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Post.php</br>Opening brace should be on a new line